include/matx/transforms/matmul/matmul_cusparse.h

```diff
@@ -80,6 +80,11 @@ class MatMulCUSPARSEHandle_t {
   using TB = typename TensorTypeB::value_type;
   using TC = typename TensorTypeC::value_type;
 
+  // Mixed-precision compute type.
+  using TCOMP = std::conditional_t<
+    std::is_same_v<TC, matx::matxFp16> ||
```

Comment:
There's a type trait for this: `is_matx_half_v`

Reply:
nice, and changed